### ISSUE TITLE
Display bug when posting certain content

### ISSUE THREAD

**Issue by viralpoet** [mastodon/mastodon#20816]
### Steps to reproduce the problem

users posting content from wordle are seeing the formatting get stripped from version 4.0.0 instances. 

![image](https://user-images.githubusercontent.com/17892651/202007557-8be98ce4-769c-4cc6-8491-6fda64785c76.png)

poeple on lower versions and those using apps, see the content properly, basically each line of 5 squares should be above each other in a block formation.

### Expected behaviour

basically each row of 5 squares should be a new row

### Actual behaviour

the formatting is stripped meaning the rows show as one line

### Specifications

Mastodon 4.0.2 - also noted on 4.0.0
Ruby 3.0.4p208
PostgreSQL 14.5
Redis 6.0.16

**Comment by ClearlyClaire** [comment]
Could you share one such post?

**Comment by ClearlyClaire** [comment]
I'm unable to reproduce it, so having an actual post example would help. That being said, this sounds like a bug that was introduced in 4.0.0 and fixed in 4.0.1, but you listed Mastodon 4.0.2 as affected so I am not sure.

**Comment by viralpoet** [comment]
I've just tried again and I can confirm its working in 4.0.2 
Not sure what I did earlier to get the same result across both but all is well

**Comment by ClearlyClaire** [comment]
ok, thank you for confirming, I'm going to close the issue then!
